### PR TITLE
Allow editing registration URLs in the admin

### PR DESCRIPTION
**Pull request by marcospri**
Allow editing URLs at the registration level (auth, token and JWKs). 

For #4304 

# Testing 

- Go over http://localhost:8001/admin/registration/id/1/
- Edit some URLs
- ????
- :tada:

### CODE REVIEW

File: lms/templates/admin/macros.jinja [hunk 1]
```diff
@@ -44,15 +44,12 @@
 {% endmacro %}
 
 
-{% macro registrations_fields(request, lti_registration, view_button=False) %}
```
Comment (marcospri) [inline lms/templates/admin/macros.jinja:47]: Typo on the macro name

File: lms/templates/admin/macros.jinja [hunk 2]
```diff
@@ -44,15 +44,12 @@
...
-<fieldset class="box">
```
Comment (marcospri) [inline lms/templates/admin/macros.jinja:48]: Just the fields themselves here

File: lms/templates/admin/macros.jinja [hunk 3]
```diff
@@ -61,16 +58,9 @@
...
-        <div class="column is-half">
-            {{ disabled_text_field("Auth login URL", lti_registration.auth_login_url) }}
-        </div>
-        <div class="column is-half">
-            {{ disabled_text_field("Key set URL", lti_registration.key_set_url) }}
-        </div>
-    </div>
-    {{ disabled_text_field("Token URL", lti_registration.token_url) }}
-</fieldset>
+    {{ form_text_field(request, "Auth login URL", "auth_login_url", lti_registration.auth_login_url) }}
```
Comment (marcospri) [inline lms/templates/admin/macros.jinja:61]: Two columns made editing more confusing. Just one column now.

**Review by seanh — APPROVED** [review]
:+1: Looks straightforward

**Pull request merged**